src/ui/wxWidgets/MergeDlg.cpp

```diff
@@ -12,24 +12,29 @@
 
 #include <wx/wxprec.h>
 
+#ifdef __BORLANDC__
```

Comment:
Where did this come from? I don't think Borland C has been around for years (RIP).